src/plugins/workspace/server/plugin.ts

```diff
@@ -78,6 +87,12 @@ export class WorkspacePlugin implements Plugin<WorkspacePluginSetup, WorkspacePl
     );
     this.proxyWorkspaceTrafficToRealHandler(core);
 
+    core.savedObjects.addClientWrapper(
+      -2,
```

Comment:
what does this -2 do? can we give it more meaningful var name?  https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/DEVELOPER_GUIDE.md#avoid-magic-numbersstrings

Reply:
Sure, let me assign it to a const and add some comment on that.